src/cloudai/schema/test_template/jax_toolbox/slurm_command_gen_strategy.py

```diff
@@ -272,6 +272,9 @@ def _generate_python_command(
                 fdl_key = cmd_arg.replace(f"{self.test_name}.fdl.", "")
                 fdl_args[fdl_key] = cmd_args[cmd_arg]
 
+        fake_flags = {"ep", "tp", "fsdp", "dp"}
```

Comment:
We have the same code in tests. Is it to exclude new fields from the CMD? If we don't want these fields there, let's exclude them like this:
```py
    ep: Optional[Union[int, list[int]]] = Field(exclude=True, default=None)
```

What it does:
1. `obj.ep` is always accessible and works as we expect.
2. During `.model_dump()` (this is what we are doing in `.cmd_args_dict`) fields marked as "exclude" will not be exported.
3. This will allow us removing duplicating code that otherwise can lead to bugs.